model/registry/file.yaml

```diff
@@ -36,3 +131,17 @@ groups:
         brief: >
           File size in bytes.
         stability: experimental
+      - id: target_path
+        type: string
+        brief: >
+          Path to the target of a symbolic link.
+        note: >
+          This attribute is only applicable to symbolic links.
+        stability: experimental
+        examples: ['/usr/bin/python3']
+      - id: uid
```

Comment:
Since it's a property of the file owner, the `file.owner.id` would be more descriptive

Reply:
Thanks for your comments. As we can't use owner as namespace as it exist already as leaf node in ECS, what do you think about 
`file.user.id` and `file.user.name` and possibly replace them later with user namespace?

Reply:
> As we can't use owner as namespace as it exist already as leaf node in ECS

I'm not sure I understand the concern. What would be the problem? There is nothing in the OTel that would prevent it happening such things in the future.

Reply:
We do have a rule [about naming](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#merging-existing-ecs-conventions) - `Do not use an existing ECS name as a namespace. ` 

In this case as there is already existing `file.owner` string field, we can't change `file.owner` to be a namespace

Reply:
Do we leave it as is or you want to change the structure and extract id and name to dedicated `user` namespace?

Reply:
We have discussed it [here](https://github.com/open-telemetry/semantic-conventions/pull/333#issuecomment-1778789854) while introducing this change and starting from [this comment](https://github.com/open-telemetry/semantic-conventions/pull/203#issuecomment-1659390589) 
So in this case we shouldn't create a namespace `file.owner.*` out of existing `file.owner` leaf field.

Reply:
@lmolkova could you check this thread?